app/controllers/stash_datacite/publications_controller.rb

```diff
@@ -77,8 +77,10 @@ def save_form_to_internal_data
         @msid = manage_internal_datum(identifier: @se_id, data_type: 'manuscriptNumber', value: parsed_msid)
       end
 
-      # if the newly-set journal wants PPR by default, set the PPR value for this resource
-      @resource.update(hold_for_peer_review: @se_id.journal&.default_to_ppr)
+      if @resource.identifier.allow_review? && @resource.previous_curated_resource.nil? && @resource.curation_start_date.nil?
```

Comment:
do we know these can only be `nil`? Maybe `blank?` would be a safer option here
```suggestion
      if @resource.identifier.allow_review? && @resource.previous_curated_resource.blank? && @resource.curation_start_date.blank?
```

Reply:
This is the same check done on the screen for the user (seen in the ticket), where it tests correctly. It seems safest for these to match.

https://github.com/CDL-Dryad/dryad-app/blob/3ad044f3eea08b14214d77a3410fccdf20d7165e/app/views/stash_datacite/peer_review/_review.html.erb#L1